docs/assets/scss/_taxonomy.scss

```diff
@@ -0,0 +1,333 @@
+// Taxonomies - e.g. Tags, Categories, ...
```

Comment:
I did not create this file, it was brought over from Hugo's base set, just like the rest of the styling files that we have, it's just that it wasn't brought over when we initially built the site out.